datanode/gateway/graphql/schema.graphql

```diff
@@ -2279,6 +2286,63 @@ type Order {
   liquidityProvision: LiquidityProvision
 }
 
+"An order update in Vega, if active it will be on the order book for the market"
```

Comment:
If an order is non-persistent, does that mean it wouldn't come up in this query? (Asking because of the description, which says "if active it will be on the order book")

Reply:
yea it should come here as well.